.gitignore

```diff
@@ -0,0 +1,2 @@
+.*.swp
```

Comment:
if you put these into your global gitignore you don't have to add them to every project